doc/Reference/Markup/StaticAndThemeResources.md

```diff
@@ -0,0 +1,156 @@
+---
+uid: Reference.Markup.StaticAndThemeResources
+---
+
+# Using Static and Theme Resources
+
+Using Static or Theme Resources is built on top of the `IDependencyPropertyBuilder<T>`. This allows the greatest degree of flexibility and extensibility.
+
+To get started let's assume that we have a string resource called `AppTitle` in our ResourceDictionary. This could be either on the element itself, somewhere within the Visual Tree or even within the Application Resources. In this case we might provide the value like:
```

Comment:
```suggestion
To get started let's assume that we have a string resource called `AppTitle` in our `ResourceDictionary`. This could be either on the element itself, somewhere within the Visual Tree or even within the `Application.Resources`. In this case we might provide the value like:
```